src/test/kotlin/com/hibob/academy/dao/OwnerDaoTest.kt

```diff
@@ -26,53 +26,36 @@ class OwnerDaoTest @Autowired constructor(private val sql: DSLContext){
-        val expected = listOf(OwnerData(ownerId, ownerName, ownerCompanyId, ownerEmployeeId))
-        assertEquals(expected, ownerDao.getOwner(ownerCompanyId))
+        val owner = OwnerCreationRequest("Noy", companyId, "123")
+        val ownerId = ownerDao.createOwner(owner)
+        val expected = listOf(OwnerData(ownerId, owner.name, owner.companyId, owner.employeeId))
+        assertEquals(expected, ownerDao.getAllOwners(companyId))
     }
 
     @Test
     fun `create a new owner that exists in the database`() {
-        val ownerName1 = "Noy"
-        val ownerCompanyId1 = companyId
-        val ownerEmployeeId1 = "123"
-
-        ownerDao.createOwner(ownerName1, ownerCompanyId1, ownerEmployeeId1)
-        val ownerName2 = "Tom"
-        assertThrows<RuntimeException> { ownerDao.createOwner(ownerName2, ownerCompanyId1, ownerEmployeeId1) }
+        val owner = OwnerCreationRequest("Noy", companyId, "123")
+        ownerDao.createOwner(owner)
+        val owner2 = OwnerCreationRequest("Tom", companyId, "123")
+        assertThrows<RuntimeException> { ownerDao.createOwner(owner2) }
     }
 
     @Test
     fun `get owner information by pet id`(){
-        val ownerName = "Noy"
-        val ownerCompanyId = companyId
-        val ownerEmployeeId = "123"
-        val ownerId1 = ownerDao.createOwner(ownerName, ownerCompanyId, ownerEmployeeId)
-
-        val petName = "Angie"
-        val petTypeString = "Dog"
-        val petCompanyId = companyId
-        val dateOfArrival = Date.valueOf("2010-05-20")
-        val petId = petDao.createPet(petName, petTypeString, petCompanyId, dateOfArrival, ownerId1)
-
-        val expected = OwnerData(ownerId1, ownerName, ownerCompanyId, ownerEmployeeId)
-        val actual = ownerDao.getOwnerByPetId(petId)
+        val owner = OwnerCreationRequest("Noy", companyId, "123")
+        val ownerId = ownerDao.createOwner(owner)
+        val pet = PetCreationRequest("Angie", convertStringToPetType("Dog"), companyId, Date.valueOf("2010-05-20"), ownerId)
+        val petId = petDao.createPet(pet)
+        val expected = OwnerData(ownerId, owner.name, owner.companyId, owner.employeeId)
+        val actual = ownerDao.getOwnerByPetId(petId, companyId)
         assertEquals(expected, actual)
     }
 
     @Test
     fun `try get information of owner by pet id for a pet that doesnt have an owner`(){
-
-        val petName = "Angie"
-        val petTypeString = "Dog"
-        val petCompanyId = companyId
-        val dateOfArrival = Date.valueOf("2010-05-20")
-        val petId = petDao.createPet(petName, petTypeString, petCompanyId, dateOfArrival, null)
-
-        val actual = ownerDao.getOwnerByPetId(petId)
+        val pet = PetCreationRequest("Angie", convertStringToPetType("Dog"), companyId, Date.valueOf("2010-05-20"), null)
+        val petId = petDao.createPet(pet)
+        val actual = ownerDao.getOwnerByPetId(petId, companyId)
         assertEquals(null, actual)
```

Comment:
this is ok - but just so you'll know - there is also `assertNull` without comparing